package.json

```diff
@@ -24,38 +24,38 @@
     "check-examples": "pnpm run --parallel --aggregate-output --reporter append-only --filter './examples/**' --filter './demos/**' check"
   },
   "dependencies": {
-    "@changesets/cli": "^2.27.5",
-    "@nucypher/nucypher-core": "^0.14.4",
-    "ethers": "^5.7.2"
+    "@changesets/cli": "^2.28.1",
+    "@nucypher/nucypher-core": "^0.14.5",
+    "ethers": "^5.8.0"
   },
   "devDependencies": {
     "@skypack/package-check": "^0.2.2",
-    "@types/node": "^20.14.2",
+    "@types/node": "^20.17.28",
     "@typescript-eslint/eslint-plugin": "^6.21.0",
     "@typescript-eslint/parser": "^6.21.0",
-    "@vitest/coverage-v8": "^1.6.0",
+    "@vitest/coverage-v8": "^1.6.1",
     "bundlemon": "^2.1.0",
-    "eslint": "^8.57.0",
+    "eslint": "^8.57.1",
     "eslint-config-prettier": "^9.1.0",
     "eslint-config-typestrict": "^1.0.5",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-import": "^2.29.1",
-    "eslint-plugin-no-only-tests": "^3.1.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-no-only-tests": "^3.3.0",
     "eslint-plugin-simple-import-sort": "^10.0.0",
     "eslint-plugin-unused-imports": "^3.2.0",
-    "gh-pages": "^6.1.1",
+    "gh-pages": "^6.3.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^3.4.2",
+    "prettier": "^3.5.3",
     "prettier-plugin-organize-imports": "^3.2.4",
-    "sort-package-json": "^2.10.0",
+    "sort-package-json": "^2.15.1",
     "ts-node": "^10.9.2",
     "ts-unused-exports": "^10.1.0",
     "typedoc": "^0.25.13",
     "typedoc-plugin-coverage": "^2.2.0",
     "typedoc-plugin-missing-exports": "^2.3.0",
-    "typedoc-plugin-zod": "^1.1.2",
-    "typescript": "^5.4.5",
-    "vitest": "^1.6.0"
+    "typedoc-plugin-zod": "^1.4.0",
+    "typescript": "^5.8.2",
+    "vitest": "^1.6.1"
```

Comment:
What about upgrading and unifying this as in `test-utils` package: `"vitest": "^3.0.9"`?

Reply:
✔️